python/snewpy/models/ccsn_loaders.py

```diff
@@ -273,7 +274,7 @@ def __init__(self, filename, metadata={}):
 
         # Get grid of model times.
         simtab['TIME'] = simtab['Tpb[ms]'] << u.ms
-        for f in [Flavor.NU_E, Flavor.NU_E_BAR, Flavor.NU_X]:
+        for f in [Flavor.NU_E, Flavor.NU_E_BAR, TwoFlavor.NU_X]:
```

Comment:
Effectively, we weren’t using these Flavors as enums here, but only using `f.name` as a hardcoded string for lookup in the files; and if we were to change `f.name` the code would break. So it’s probably more honest to use the strings directly.